packages/json/json.pony

```diff
@@ -75,7 +85,17 @@ class JsonObject
         text = text + ","
       end
 
-      text = text + "\n" + elem_indent + "\"" + i._1 + "\": " +
-        _JsonPrint._string(i._2, elem_indent)
+      if pretty_print then
+        text = text + "\n" + elem_indent + "\"" + i._1 + "\": " +
+          _JsonPrint._string(i._2, elem_indent, true)
+      else
+        text = text + if text.size() > 0 then " " else "" end
```

Comment:
This entire conditional expression can be removed to remove the space-after-comma to save extra bytes when not-pretty printing.